pyuntl/form_logic.py

```diff
@@ -353,12 +362,12 @@ def __init__(self, **kwargs):
 
 class Identifier(FormElement):
     def __init__(self, **kwargs):
```

Comment:
This may _eventually_ fail updated flake8 checks because PEP-8 was updated last year to indicate that methods should be _surrounded_ by single blank lines rather than _separated_ by single blank lines (pep8 hasn't updated their check to include this modification yet). I personally think the blank line isn't needed when there is only one method and no docstring, but I just wanted to point it out.